src/test/clone/CMakeLists.txt

```diff
@@ -38,6 +38,8 @@ add_shadow_tests(
     ARGS --strace-logging-mode=off
 )
 
+find_program(PYTHON3 python3foob)
```

Comment:
comment?

Reply:
Oops, deleted.

I was starting to sketch out locating the python binary here and dynamically generating the sim config to pass in that path as a command-line argument, instead of hard-coding `/usr/bin/python3` in the tests, but I'm still thinking about whether that's the way to go. In any case will leave for a follow-up.